src/test/java/com/checkmarx/flow/cucumber/integration/sca_scanner/teams/SCATeamsSteps.java

```diff
@@ -0,0 +1,126 @@
+import org.apache.http.HttpStatus;
+import org.junit.Assert;
+import org.springframework.boot.test.context.SpringBootTest;
+
+import java.io.IOException;
+import java.rmi.Remote;
+import java.util.Collections;
+import java.util.List;
+
+@SpringBootTest(classes = {CxFlowApplication.class})
+@Slf4j
+public class SCATeamsSteps extends ScaCommonSteps {
+
+    private static final String PROJECT_NAME = "Team-Test-Project";
+    private final ScaProperties scaProperties;
+
+    private ScaClientHelper scaClientHelper;
+    private String projectId;
+    private int errorStatusCode;
+
+    public SCATeamsSteps(FlowProperties flowProperties, SCAScanner scaScanner,
+                         ScaConfigurationOverrider scaConfigOverrider, ScaProperties scaProperties) {
+        super(flowProperties, scaScanner, scaConfigOverrider);
+        this.scaProperties = scaProperties;
+        this.scaClientHelper = new ScaClientHelper(createRestClientConfig(), log, scaProperties);
+    }
+
+    @After()
+    public void cleanUp() throws IOException {
+        if (StringUtils.isNotEmpty(projectId)) {
+            scaClientHelper.deleteProjectById(projectId);
+        }
+    }
+
+    @Before()
+    public void init() throws IOException {
+        initSCAConfig(scaProperties);
+        scaClientHelper.init();
+        deleteProjectIfAlreadyExists();
+    }
+
+    @Given("scanner is SCA")
+    public void setScanner() {
+        flowProperties.setEnabledVulnerabilityScanners(Collections.singletonList(ScaProperties.CONFIG_PREFIX));
+    }
+
+    @When("creating a new project with associated {string} value")
+    public void createNewProjectTeam(String team) throws IOException {
+        if (team.equals("null")) {
+            team = null;
+        }
+        scaProperties.setTeamForNewProjects(team);
+        try {
+            projectId = scaClientHelper.createRiskManagementProject(PROJECT_NAME);
+        } catch (CxHTTPClientException e) {
+            errorStatusCode = e.getStatusCode();
+        }
+    }
+
+    @Then("project assignedTeams returned value is {string}")
+    public void validateAssignedTeam(String expectedAssignedTeam) throws IOException {
+        String actualTeam;
+        Project projectDetails = scaClientHelper.getProjectDetailsByProjectId(projectId);
+        List<String> assignedTeams = projectDetails.getAssignedTeams();
+
+        if (assignedTeams.isEmpty()) {
+            actualTeam = "";
+            actualTeam = "";
+        } else {
+            actualTeam = assignedTeams.get(0);
+        }
+
+        Assert.assertEquals(expectedAssignedTeam, actualTeam);
+    }
+
+    @Then("bad request error is expected to be thrown")
+    public void validateNotExistsTeam() {
+        Assert.assertEquals(HttpStatus.SC_BAD_REQUEST, errorStatusCode);
+    }
+
+
+    private void deleteProjectIfAlreadyExists() throws IOException {
+        String projectId = scaClientHelper.createRiskManagementProject(PROJECT_NAME);
```

Comment:
Why do you first create a project and then delete it?

Reply:
We delete the project at the end to clean the data the test created

Reply:
Will be fixed